helm/templates/api/api-configmap.yaml

```
@@ -490,6 +490,13 @@ data:
     gateway:
       unknown-expire-after: {{ .Values.api.gateway.unknownExpireAfter }}
 
+    # Integration controller
+    integration:
+      enabled: {{ .Values.api.integration.enabled }}
+      controller:
+        ws:
+          port: {{ .Values.api.integration.port }}
+
```

```suggestion
    # Integration controller
    {{- if .Values.api.integration.enabled }}
    integration:
      enabled: {{ .Values.api.integration.enabled }}
      controller:
        ws:
          port: {{ .Values.api.integration.port }}
    {{- end }}
```
Maybe not include this section is not enable ?

---

I will keep 

```
integration:
  enabled: {{}}
```

However I won't include the rest if disabled